src/compounds/ActionsGroup/ActionsGroup.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { ListGroup } from 'react-bootstrap'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import SendMessage from './actions/SendMessage'
+import './actions-group.scss'
+
+const ActionsGroup = ({ handleSendingMessages }) => {
```

Comment:
as we add new actions, they each will need to pass in a corresponding thing to do when clicked.